config/environments/appliance.rb

```diff
@@ -0,0 +1,67 @@
+Rails.application.configure do
+  # Settings specified here will take precedence over those in config/application.rb.
+
+  # Code is not reloaded between requests.
+  config.cache_classes = true
+
+  # Eager load code on boot. This eager loads most of Rails and
+  # your application in memory, allowing both threaded web servers
+  # and those relying on copy on write to perform better.
+  # Rake tasks automatically ignore this option for performance.
+  config.eager_load = true
+
+  # Full error reports are disabled and caching is turned on.
+  config.consider_all_requests_local       = false
+  config.action_controller.perform_caching = true
+
+  # Enable Rack::Cache to put a simple HTTP cache in front of your application
+  # Add `rack-cache` to your Gemfile before enabling this.
+  # For large-scale production use, consider using a caching reverse proxy like
+  # NGINX, varnish or squid.
+  # config.action_dispatch.rack_cache = true
+
+  # Disable serving static files from the `/public` folder by default since
+  # Apache or NGINX already handles this.
+  config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
+
+  # Use log level "warn" in prod to avoid logging parameters.
+  config.log_level = :warn
+
+  # Specifies the header that your server uses for sending files.
+  # config.action_dispatch.x_sendfile_header = 'X-Sendfile' # for Apache
+  # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
+
+  # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
+  # config.force_ssl = true
+
+  # Prepend all log lines with the following tags.
+  # config.log_tags = [ :subdomain, :uuid ]
+
+  # Use a different logger for distributed setups.
+  # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
+
+  # Use a different cache store in production.
+  # config.cache_store = :mem_cache_store
+
+  # Enable serving of images, stylesheets, and JavaScripts from an asset server.
+  # config.action_controller.asset_host = 'http://assets.example.com'
+
+  # Ignore bad email addresses and do not raise email delivery errors.
+  # Set this to true and configure the email server for immediate delivery to raise delivery errors.
+  # config.action_mailer.raise_delivery_errors = false
+
+  # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
+  # the I18n.default_locale when a translation cannot be found).
+  config.i18n.fallbacks = true
+
+  # Send deprecation notices to registered listeners.
+  config.active_support.deprecation = :notify
+
+  config.log_level = :info
+  config.middleware.use Audit::RememberUuid
+  config.audit_socket = '/run/conjur/audit.socket'
```

Comment:
How would I enable audit using in `RAILS_ENV=production`? For authn-local, the audit socket location is basically hard-coded and the service just starts if the socket exists. Any thoughts on the benefits of one approach vs the other?

Reply:
With `RAILS_ENV=production` as with the other environments the audit messages go to the normal log sink (formatted with that sink's formatter). 

This is 'hardcoded' here specifically for `appliance` environment which is what the environment is set to in the appliance, which seems appropriate; this is a configuration file after all. Perhaps this configuration file should go in the appliance instead of this repository, but I thought it'd make it harder to manage (and also that's how we had it in v4).

Reply:
But, if I am using the open source edition, and I want to get the audit messages, I want to be able to add a setting to enable the audit without having to modify a source file. And I want to run with `RAILS_ENV=production`